examples/Shop/CreateProductJsonOrderwithSignature.php

```diff
@@ -65,20 +60,18 @@ function printJsonOrder($parameterCacheId, $productsTotalPrice)
  * @return string
  *
  */
-
 function jsonEncode($pData)
 {
 	return json_encode($pData, JSON_UNESCAPED_UNICODE|JSON_UNESCAPED_LINE_TERMINATORS);
 }
 
 /**
- *
  * @param string $jsonData
  * @param string $secret
  * @return string
  *
+ * @throws Exception
```

Comment:
internally we do not set throws to DocBlocks. 
Do we handel it in another way for git?
Removing empty blocks and the other changes are great. I like it!

Reply:
Yes, we do it internally, because we have some company rules that enforce us to do this behaviour. But we are developing open source, which means a repository should always be easy to use. And easy to use also means that we define what could happen.